content/index.md

```diff
@@ -45,7 +45,12 @@
       <li>
         <div class="schedule-entry flex gap-4 mb-4 border-b-2 border-b-stone-900 border-dashed pb-4">
           <time class="twokay-font-freshman uppercase w-16 text-right shrink-0">12:30</time>
-          <div class="grow-0">Lunch Break</div>
+          <div class="grow-0">
+            <a class="block twokay-font-freshman uppercase mb-1">Lunch Break</a>
+            <span class="block text-xl leading-none ">
+              Provided by <a class="underline font-black" href="https://rgfoods.net/home" target="_blank">RG Foods</a>
+            </span>
+          </div>
```

Comment:
Note: The [issue](#334) requested adding the day-of menu. That was not feasible at this time.

Reply:
Nevermind! Added it to the commented out FAQs section. 👍